### ISSUE TITLE
[BUG] Immutable -- Installer did not reclaim hard drive space

### ISSUE THREAD

**Issue by Linux-Is-Best** [Ultramarine-Linux/ultramarine#24]
**Describe the bug**

During the installation, I selected both hard drives, and left the system to decide how best to partition the drives, being an immutable distro.  Usually, for example, Fedora Silver Blue or Universal Blue (to compare), it would ask me if I wanted to reclaim the whole drive.  I was not presented with that choice, and now find I can dual-boot between my Immutable OS (Ultramarine) and Windows 11.  I had plans or expected it to wipe the whole drive.

**To Reproduce**
1. Boot from the ISO
2. Select multiple drives
3. Keep the auto settings
4. Continue the installation 

**Expected behavior**

Being an immutable, I expected it to wipe the drives, and not attempt to save Windows. It's cool that it could, but it was unexpected. 

**Screenshots**

N / A

**Desktop (please complete the following information):**
 - OS: Ultramarine Linux Flagship Edition -- Immutable
 - Version:

ostree-unverified-image:docker://ghcr.io/ultramarine-linux/ultramarine-flagship:39
Digest: sha256:558022c52632c00fea243de55b5db0abc185319bd7d089de813ce6292e701b01
Timestamp: 2023-11-25T06:08:46Z

**Additional context**

If this is something you want, it should ask what the user intention is, when selecting both drives.

**Comment by nothingneko** [comment]
this may be a KS issue, I'll try to look into it, we'll also keep the feedback in mind when designing the new installer

**Comment by Linux-Is-Best** [comment]
This needs to be re-verified.  

I could reproduce this issue twice before, but with the new ISO generated today (build 107) I could not reproduce the issue. 

In my recent test, I had the choice of checking the box to reclaim space and this time it did indeed wipe the drive. Or I could choose not to check anything and dual boot. 

@nothingneko did you or someone fix this?

**Comment by Linux-Is-Best** [comment]
I am closing the issue.

Why?

Except for those 2x when I experience it, I cannot reproduce the issue in the last couple of beta builds. I have installed and reinstall both on physical hardware and in a virtual machine. I have successfully been given the choice to reclaim my hard drive space, and it has done so when selected. 

I don't know if it was a limited issue in only the previous build, or I had a bad ISO burn. But the fact that no one else has encountered this issue, plus the fact that I allowed an automated script to repeatedly reinstall via a VM (virtual machine) without issue, I take it as a good sign that this is not an issue.  

(My little bot reinstall the OS in a VM 400x without issue).
